iam/api/QuickStartNewTest/QuickStartNewTest.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using Google.Apis.Auth.OAuth2;
+using Google.Apis.Iam.v1;
+using Google.Apis.Iam.v1.Data;
+using Xunit;
+
+public class QuickStartNewTest : IDisposable
+{
+    private readonly string _projectId = Environment.GetEnvironmentVariable("GOOGLE_PROJECT_ID");

```

Comment:
Consider doing this in the constructor and checking that is not empty. You can throw an exception if it is empty. It helps a lot when running locally to get that exception if you have forgotten to set then env variable.